CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 
 - Added sensitivity calculation support for `pybamm.Simulation` and `pybamm.Experiment` ([#4415](https://github.com/pybamm-team/PyBaMM/pull/4415))
 - Added OpenMP parallelization to IDAKLU solver for lists of input parameters ([#4449](https://github.com/pybamm-team/PyBaMM/pull/4449))
-- Added phase-dependent particle options to LAM #4369
+- Added phase-dependent particle options to LAM
+  ([#4369](https://github.com/pybamm-team/PyBaMM/pull/4369))
```

Comment:
Fixed this one while I was updating my own log entry

Reply:
Should it not be in a single line? 😕

Reply:
The wrapping was from my IDE, it does not make a difference in markdown as long as the link is not split between lines

Reply:
I see, it's fine in that case.